# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/redis/remove-version-check.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- redis-6.2.6/src/zmalloc.h.vanilla	2022-02-19 12:46:19.124000000 +0200
+++ redis-6.2.6/src/zmalloc.h	2022-02-19 12:46:49.114000000 +0200
@@ -48,12 +48,7 @@
 #elif defined(USE_JEMALLOC)
 #define ZMALLOC_LIB ("jemalloc-" __xstr(JEMALLOC_VERSION_MAJOR) "." __xstr(JEMALLOC_VERSION_MINOR) "." __xstr(JEMALLOC_VERSION_BUGFIX))
 #include <jemalloc/jemalloc.h>
-#if (JEMALLOC_VERSION_MAJOR == 2 && JEMALLOC_VERSION_MINOR >= 1) || (JEMALLOC_VERSION_MAJOR > 2)
-#define HAVE_MALLOC_SIZE 1
 #define zmalloc_size(p) je_malloc_usable_size(p)
-#else
-#error "Newer version of jemalloc required"
-#endif
 
 #elif defined(__APPLE__)
 #include <malloc/malloc.h>
